lib/src/base-styles/_nx-tile.scss

```diff
@@ -45,10 +42,9 @@ $-tile-header-height-actions: 43px; // Height with sub-title and/or right action
   // NOTE: without the min-content sizing on the rows it adds extra vertical space when only title and actions are
```

Comment:
Since we're getting rid of one of the two `min-content` usages, I'm not 100% sure this comment still fully applies.

Reply:
> Since we're getting rid of one of the two `min-content` usages, I'm not 100% sure this comment still fully applies.

Thanks, removed it here: 8cf12124868d92ed59719abfba159faadb763ff4